app/(authed)/dashboard/_components/sidebar.tsx

```
@@ -25,14 +19,14 @@ import {
   SidebarMenuSubItem,
   SidebarRail,
 } from "@/components/ui/sidebar";
+import UserMenu from "@/components/user-menu";
 import { ROUTES } from "@/lib/constants";
-import { SignOutButton } from "@clerk/nextjs";
 import { currentUser } from "@clerk/nextjs/server";
 import Link from "next/link";
-import ManageAccountButton from "./manage-account-button";
 
 export default async function DashboardSidebar() {
   const user = await currentUser();
```

ESLint

[error] 'user' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)